themes/10up-theme/assets/css/frontend/base/wordpress.css

```diff
@@ -51,7 +51,9 @@
 	@apply max-w-none;
 }
 
-/**
+@layer components {
```

Comment:
@Nikki-Jones Is the intent here to make these styles below purgeable? Or are they supposed to always be in the CSS output?

Things like `.editor-post-title__input` look like a Block Editor class, which wouldn't be picked up in our content patterns since it's not in the actual theme.